commands/deals_daemon_test.go

```diff
@@ -180,27 +190,33 @@ func TestShowDeal(t *testing.T) {
 
 	ask, err := series.CreateStorageMinerWithAsk(ctx, minerNode, collateral, price, expiry)
 	require.NoError(t, err)
+	defer func() {
+		require.NoError(t, minerNode.MiningStop(ctx))
+	}()
 
 	// Create some data that is the full sector size and make it autoseal asap
 
 	maxBytesi64 := int64(getMaxUserBytesPerStagedSector())
 	dataReader := io.LimitReader(rand.Reader, maxBytesi64)
+
+	// Duration is 10
```

Comment:
It's not clear what this comment is referring to or why it's needed.

Reply:
That was a note to myself when I was trying to do some debugging. I'll take it out.  It's the default duration for ImportAndStore.